### DISCUSSION TITLE
Knapper med piler

### DISCUSSION THREAD

**Discussion by kennidenni** [fremtind/jokul#2768] [Annet]
Vi i PM skal nå ha piler i knapper for å gi en ekstra indikasjon for å gå fremover og tilbake. 

Er det noen andre som bruker det og er det noe som skulle vært mulig å ha på knappene som et flagg?

![image](https://user-images.githubusercontent.com/31621649/156157901-f2e366bb-7278-44d9-a252-43f209ebdc96.png)

**Comment by wkillerud** [comment]
Da ble vi enige om på forum i dag å ta det inn i biblioteket blant annet for å sørge for riktig skjermleseropplevelse og spacing rundtomkring.

**Reply by wkillerud** [reply]
Jeg laget #2772. @kennidenni, er det noe du og Joakim kan ta på dere? 😇

**Reply by kennidenni** [reply]
Jepp, kan det!
